src/core/server/rendering/rendering_service.tsx

```diff
@@ -63,18 +63,20 @@ export class RenderingService {
       .pipe(first())
       .toPromise();
 
+    const isFullLogoUrlValid = await this.checkUrlValid(
+      opensearchDashboardsConfig.branding.fullLogoUrl,
+      'fullLogoUrl'
+    );
     const isLogoUrlValid = await this.checkUrlValid(
       opensearchDashboardsConfig.branding.logoUrl,
       'logoUrl'
     );
-    const isSmallLogoUrlValid = await this.checkUrlValid(
-      opensearchDashboardsConfig.branding.smallLogoUrl,
-      'smallLogoUrl'
-    );
     const isLoadingLogoUrlValid = await this.checkUrlValid(
       opensearchDashboardsConfig.branding.loadingLogoUrl,
       'loadingLogoUrl'
     );
+    const isTitleValid = this.checkTitleValid(opensearchDashboardsConfig.branding.title, 'title');
+    const defaultTitle = 'OpenSearch Dashboards';
```

Comment:
this probably too nested, in my opinion the default value for title should either keep it in the opensearch_dashboards_config file or at least pull it out of this class as DEFAULT_TITLE

Reply:
I thought about keeping the default value in the opensearch_dashboards_config file. I choose not to go this route because if user inputs an empty string, the default value "OpenSearch Dashboards" from the config file will be passed to rendering service. Then it will actually pass the title validation function and will not show the log warning " title is invalid or not found" while an empty string input is actually an invalid input. 

I also agree that we should not keep this default value too nested since outside user may want to reference it in some cases in the future. I will pull this out of class and have a const DEFAULT_TITLE = "OpenSearch Dashboards" for now. Thx!